sites/kit.svelte.dev/src/routes/+page.svelte

```diff
@@ -1,12 +1,22 @@
 <script>
+	import { onMount } from 'svelte';
 	import Features from './home/Features.svelte';
 	import Hero from './home/Hero.svelte';
 	import Showcase from './home/Showcase.svelte';
 	import Try from './home/Try.svelte';
 	import Deployment from './home/Deployment.svelte';
 	import Svelte from './home/Svelte.svelte';
 	import Intro from './home/Intro.svelte';
+	import schema_url from './schema.json?url';
 	import './home/common.css';
+
+	let schema;
+	onMount(async () => {
+		// Google will not allow linking to a schema file. It must be in the DOM
+		// We load it and add it to the DOM to save bytes on page load vs inlining
+		const json = await fetch(schema_url).then((r) => r.text());
```

Comment:
@Rich-Harris could this also be written as:
```suggestion
		const json = await (await fetch(schema_url)).text();
```